src/main/java/stsjorbsmod/cards/cull/Godsbane.java

```diff
@@ -0,0 +1,53 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+
+public class Godsbane extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Godsbane.class);
+
+    private static final CardRarity RARITY = CardRarity.COMMON;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 1;
+    private static final int DAMAGE_PERCENT = 8;
+    private static final int UPGRADE_DAMAGE_PERCENT = 2;
+
+    public Godsbane() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        magicNumber = baseMagicNumber = DAMAGE_PERCENT;
+        damage = baseDamage = 0;
+    }
+
+    private void recalculateDamage(AbstractMonster m) {
+        damage = (int) Math.round((m.maxHealth * ((double) magicNumber / 100)));
+    }
+
+    @Override
+    public void calculateCardDamage(AbstractMonster m) {
+        super.calculateCardDamage(m);
```

Comment:
I am actually being dumb here. What's the thought behind calling the super? I'm guessing it's in the general case that the super has some side effects beyond damage calculation?